features/step_definitions/api/application_plans_steps.rb

```diff
@@ -10,6 +10,23 @@
   assert_equal @plan, default_service.reload.default_application_plan
 end
 
+When "the default application plan is set" do
+  plan = FactoryBot.create(:application_plan, issuer: default_service)
+  default_service.default_application_plan = plan
+  default_service.save!
+end
+
+When "an admin unsets the default application plan" do
+  visit admin_service_application_plans_path(default_service)
+  select_default_plan_by_name "No plan selected"
+end
+
+Then "the service will not have the default plan set" do
+  visit admin_service_application_plans_path(default_service)
+  sleep 5
```

Comment:
Neither of these should be necessary

Reply:
Oh, the `sleep` is obviously is just a debugging leftovers... will remove it.
In this test I wanted specifically to test the UI, not just the DB value.